submissions/t-jay-777/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Hooli-style Popup</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+       <div class="container">
+        <div class="nav">
```

Comment:
For navigation better to use `nav` tag, it's more semantic and helps to recognize the role of different parts of your page.

Reply:
> For navigation better to use `nav` tag, it's more semantic and helps to recognize the role of different parts of your page.

Do you mean main menu only or submenu also?

Reply:
I mean for full nav block
Example: replace `div.nav` by `nav.nav`

Reply:
> For navigation better to use `nav` tag, it's more semantic and helps to recognize the role of different parts of your page.

Do you mean main menu only or main manu + submanu also?

Reply:
It depends on the design some times its main menu + nested submenu, sometimes it's the only main menu. A good way to choose the right HTML tag is to look at its purpose.
In this case, we have only one nav block.